.cirrus.yml

```diff
@@ -51,15 +51,18 @@ FreeBSD_task:
     - ${PY} -m pip list
     - ./test.sh
 
-Linux_task:
+task:
   skip: $CIRRUS_BRANCH =~ '.*\.tmp'
-  allow_failures: $CIRRUS_TASK_NAME =~ '.*-rc-.*'
-  container:
+  env:
     matrix:
-      - image: python:3.6-slim
-      - image: python:3.7-slim
-      - image: python:3.8-rc-slim
-      - image: pypy:3.6-slim
+      - IMAGE_TAG: 3.6-slim
+      - IMAGE_TAG: 3.7-slim
+      - IMAGE_TAG: 3.8-rc-slim
+      - IMAGE_TAG: 3.6-slim
```

Comment:
Dropping the image name created some confusion between `python:3.6-slim` and `pypy:3.6-slim`.
(The former uses the CPython interpreter, the latter uses the PyPy JIT-compiled VM)

Fixing that then merging.